src/main/java/com/notably/model/block/Title.java

```diff
@@ -0,0 +1,64 @@
+package com.notably.model.block;
+
+import static com.notably.commons.util.AppUtil.checkArgument;
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents a Block's title in Notably.
+ * Guarantees: immutable; is valid as declared in {@link #isValidTitle(String)}
+ */
+public class Title {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Titles should only contain alphanumeric characters and spaces, and it should not be blank";
+
+    /*
+     * The first character of the title must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
```

Comment:
This regex can be rewritten as `"^\\p{Alnum}[\\p{Alnum} ]*$"`.
The `[]` in the first `\\p{Alnum}` isn't really necessary, as it is only a single item.
The `^$` symbol anchors the regex to the start and end of the string (whole string)

Reply:
Actually, after reading more about how `String#matches` work, I think the `^$` part is not necessary

Reply:
Changed! :)